src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/missingjavadocmethod/InputMissingJavadocMethodScopeAnonInner.java

```diff
@@ -8,52 +8,54 @@
 import java.awt.event.MouseAdapter;
 import javax.swing.JButton;
 
+/* Config:
+ * scope = Scope.PRIVATE.getName()
+ */
 /**
  * Tests for anonymous inner types
- * @author Lars Kühne
- **/
+ */
 public class InputMissingJavadocMethodScopeAnonInner
 {
     /**
-       button.
-    */
+     * button.
+     */
     private JButton mButton = new JButton();
 
     /**
-       anon inner in member variable initialization.
-    */
-    private Runnable mRunnable = new Runnable() {
-        public void run() // should not have to be documented, class is anon.
+     * anon inner in member variable initialization.
+     */
```

Comment:
Did we need to reformat these for test cases?

Reply:
No, but we might as well have similarly formatted javadoc in all input files, so I tweaked it. Let me know if this change should be reverted.

Reply:
I would keep all reformatting to separate commit or revert, it just helps reviews to go faster when there is less changes to look at, or when they are obviously atomic.

Reply:
Moved reformatting to new commit.

Reply:
Resolved.